cirq-google/cirq_google/optimizers/two_qubit_gates/gate_compilation_test.py

```diff
@@ -9,16 +9,40 @@
     gate_product_tabulation,
     GateTabulation,
 )
-from cirq_google.optimizers.two_qubit_gates.math_utils import unitary_entanglement_fidelity
-from cirq.testing import random_special_unitary, assert_equivalent_repr
+from cirq.optimizers.gate_tabulation_math_utils import unitary_entanglement_fidelity
+from cirq.testing import random_special_unitary
+
+ALLOW_DEPRECATION_IN_TEST = 'ALLOW_DEPRECATION_IN_TEST'
```

Comment:
We should only test that the remaining public objects emit a deprecation warning when they get called. The behavior specific tests should have already been moved to the new location.

Reply:
Done.